### DISCUSSION TITLE
OAuth Connection with Mailcow not working

### DISCUSSION THREAD

**Discussion by Sarsum** [requarks/wiki#2957] [Help / Questions]
Hey there,

I am currently trying to set up OAuth Login with [Mailcow](https://mailcow.github.io/mailcow-dockerized-docs/) for WikiJS.
I have set every data as described, but I am getting the error `An unsupported scope was requested` in WikiJS.
It seems that Mailcow is only supporting the scope `profile` and rejecting every request with (additional) unsupported scopes.

Is there any possibility to set the scopes WikiJS is requesting?

Host infomration:
```
OS: Debian 10 (Buster)
Docker Version: 20.10.1
Docker-Compose Version: 1.27.4
WikiJS Version: 2.5.170
PostgreSQL Version: 11.10
```

Screenshot of the error:
![firefox_yjLtDZHfAI](https://user-images.githubusercontent.com/33085157/105843264-63132480-5fcf-11eb-938e-5d2eb3c12658.png)

**Comment by DJIronic** [comment]
I have the same issue.
